README.md

```diff
@@ -79,83 +80,248 @@ Now it's your turn to write SQL queries to achieve the following results (You ne
+        ```
+        IF saldo_despues_transfer < 731823.56 THEN
+            RAISE EXCEPTION 'Saldo insuficiente para el movimiento OUT de 731823.56';
+        END IF;
+
+        INSERT INTO movements (id, account_from, account_to, type, mount, created_at)
+        VALUES (gen_random_uuid(), '3b79e403-c788-495a-a8ca-86ad7643afaf', NULL, 'OUT', 5000.00, CURRENT_TIMESTAMP);
+
+        COMMIT;
+        ```
+
+        e. How much money the account `fd244313-36e5-4a17-a27c-f8265bc46590` have:
+        ```
+            WITH user_balances AS (
+        SELECT
+            a.user_id,
+            SUM(CASE
+                    WHEN m.type = 'IN' THEN m.mount
+                    WHEN m.type = 'TRANSFER' AND m.account_to = a.id THEN m.mount
+                    WHEN m.type = 'TRANSFER' AND m.account_from = a.id THEN -m.mount
+                    WHEN m.type = 'OUT' THEN -m.mount
+                    WHEN m.type = 'OTHER' AND m.account_from = a.id THEN -m.mount
+                    WHEN m.type = 'OTHER' AND m.account_to = a.id THEN m.mount
+                    ELSE 0
+                END) + a.mount AS total_balance
+        FROM
+            accounts a
+        LEFT JOIN
+            movements m ON (m.account_from = a.id OR m.account_to = a.id)
+        GROUP BY
+            a.user_id, a.mount
+
+    )
+    SELECT total_balance
+    FROM user_balances
+    WHERE user_id = 'fd244313-36e5-4a17-a27c-f8265bc46590';
     ```
-        Your query
-    ```
-
 
-6. All the movements and the user information with the account `3b79e403-c788-495a-a8ca-86ad7643afaf`
+6.  All the movements and the user information with the account `3b79e403-c788-495a-a8ca-86ad7643afaf`
 
 ```
-Your query here
+SELECT
+    us.id AS user_id,
+    us.name AS user_name,
+	ac.id AS account_id,
+    ac.type AS account_type,
+    mov.id AS movement_id,
+	mov.type AS movement_type,
+    mov.mount,
+    mov.account_from,
+    mov.account_to
+FROM
+    movements mov
+INNER JOIN
+    accounts ac ON mov.account_from = ac.id OR mov.account_to = ac.id
+INNER JOIN
+    users us ON ac.user_id = us.id
+WHERE
+    mov.account_from = '3b79e403-c788-495a-a8ca-86ad7643afaf'
+    OR
+    mov.account_to = '3b79e403-c788-495a-a8ca-86ad7643afaf';
 ```
 
-
 7. The name and email of the user with the highest money in all his/her accounts
 
 ```
-Your query here
+SELECT
+	CONCAT (us.name, ' ', us.last_name),
+	us.email
+FROM accounts ac
+INNER JOIN users us ON ac.user_id = us.id
+GROUP BY ac.user_id, us.name, us.email, us.last_name
+ORDER BY SUM(ac.mount) DESC
+LIMIT 1;
 ```
 
-
 8. Show all the movements for the user `Kaden.Gusikowski@gmail.com` order by account type and created_at on the movements table
 
 ```
-Your query here
+SELECT
+    us.id AS user_id,
+    us.name AS user_name,
+	ac.id AS account_id,
+    ac.type AS account_type,
+    mov.id AS movement_id,
+	mov.type AS movement_type,
+    mov.mount,
+    mov.account_from,
+    mov.account_to,
+	mov.created_at
```

Comment:
since we know who is the user we're looking for, we can omit the columns about it, and keep the columns about movements: `SELECT mov.*`